cluster-autoscaler/main.go

```diff
@@ -437,3 +451,37 @@ func validateMinMaxFlag(min, max int64) error {
 func minMaxFlagString(min, max int64) string {
 	return fmt.Sprintf("%v:%v", min, max)
 }
+
+func parseMultipleGpuLimits(flags MultiStringFlag) ([]context.GpuLimits, error) {
+	parsedFlags := make([]context.GpuLimits, 0, len(flags))
+	for _, flag := range flags {
+		parsedFlag, err := parseSingleGpuLimit(flag)
+		if err != nil {
+			return nil, err
+		}
+		parsedFlags = append(parsedFlags, parsedFlag)
+	}
+	return parsedFlags, nil
+}
+
+func parseSingleGpuLimit(config string) (context.GpuLimits, error) {
```

Comment:
This function is basically duplicated in gpu.go (ParseGpuLimits). And I'm not sure if I like having it here. On one hand it is consistent with how we handle CPU/mem limits. On the other I don't like having all this code in main.

Reply:
Yeah - actually I forgot to remove the function from `gpu.go` which is not used now. It seems to me that it is valid to have a function which parses user flags around the place which defines those flags.
Should it be main.go ... I am not sure. But as we define (for now) flags here it seems to me that it is valid place.